eoscostarica/guias.eoscostarica.io
# Docs/check titles for relevance

### docs(frontend): Checking titles for relevance

### What does this PR do?

- resolves #394

### Steps to test
1. To go to [EOS Costa Rica](https://guias.eoscostarica.io/)
2. See some titles that were changed.
3. The others change can't see because they are to improve SEO

#### CheckList
